src/modules/PowerOCR/PowerOCR/Helpers/ImageMethods.cs

```diff
@@ -146,11 +147,27 @@ await using (MemoryStream memory = new MemoryStream())
                 }
                 else
                 {
+                    var cjkRegex = new Regex(@"\p{IsCJKUnifiedIdeographs}");
+
                     foreach (OcrLine ocrLine in ocrResult.Lines)
                     {
+                        bool isBeginning = true;
```

Comment:
I feel like the logic maybe could be tweaked?  Like, 
forloop
{
  if ! (isCJK && isCJKPrev)
     append space

  append word
  set isCJKPrev 
}
trim at the end of forloop

Reply:
Indeed, there's no reason for the isBeginning, since is CJKPrev is always false. It won't output a space in the beginning.

Reply:
Nevermind, the beginning logic is indeed needed. I think it's better to keep track as well (to avoid having a string rewrite where we have to shift every character)